<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatorWnd</class>
 <widget class="QMainWindow" name="CreatorWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>466</width>
         <height>390</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>619</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="closeAction"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>查看</string>
    </property>
    <addaction name="action"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="saveAction"/>
   <addaction name="separator"/>
   <addaction name="pointerAction"/>
   <addaction name="textAction"/>
   <addaction name="separator"/>
   <addaction name="boldAction"/>
   <addaction name="italicAction"/>
   <addaction name="underlineAction"/>
   <addaction name="fontAction"/>
   <addaction name="colorAction"/>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>119</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>524287</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>对象</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="margin">
      <number>0</number>
     </property>
     <item row="0" column="0">
      <widget class="QListWidget" name="listWidget"/>
     </item>
     <item row="1" column="0">
      <widget class="QToolButton" name="delButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="saveAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/filesave.png</normaloff>:/Resources/filesave.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
  </action>
  <action name="saveAsAction">
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="closeAction">
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="pointerAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/pointer.png</normaloff>:/Resources/pointer.png</iconset>
   </property>
   <property name="text">
    <string>移动工具</string>
   </property>
  </action>
  <action name="textAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textpointer.png</normaloff>:/Resources/textpointer.png</iconset>
   </property>
   <property name="text">
    <string>文本工具</string>
   </property>
  </action>
  <action name="cutAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/editcut.png</normaloff>:/Resources/editcut.png</iconset>
   </property>
   <property name="text">
    <string>剪切</string>
   </property>
  </action>
  <action name="copyAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/editcopy.png</normaloff>:/Resources/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>复制</string>
   </property>
  </action>
  <action name="pasteAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/editpaste.png</normaloff>:/Resources/editpaste.png</iconset>
   </property>
   <property name="text">
    <string>粘贴</string>
   </property>
  </action>
  <action name="delAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/editdelete.png</normaloff>:/Resources/editdelete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </action>
  <action name="selectAllAction">
   <property name="text">
    <string>全选</string>
   </property>
  </action>
  <action name="boldAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textbold.png</normaloff>:/Resources/textbold.png</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
  </action>
  <action name="italicAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textitalic.png</normaloff>:/Resources/textitalic.png</iconset>
   </property>
   <property name="text">
    <string>斜体</string>
   </property>
  </action>
  <action name="underlineAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textunder.png</normaloff>:/Resources/textunder.png</iconset>
   </property>
   <property name="text">
    <string>下划线</string>
   </property>
  </action>
  <action name="leftAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textleft.png</normaloff>:/Resources/textleft.png</iconset>
   </property>
   <property name="text">
    <string>左对齐</string>
   </property>
  </action>
  <action name="centerAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textcenter.png</normaloff>:/Resources/textcenter.png</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
  </action>
  <action name="rightAction">
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textright.png</normaloff>:/Resources/textright.png</iconset>
   </property>
   <property name="text">
    <string>右对齐</string>
   </property>
  </action>
  <action name="fontAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/textsubscript.png</normaloff>:/Resources/textsubscript.png</iconset>
   </property>
   <property name="text">
    <string>字体</string>
   </property>
  </action>
  <action name="colorAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/Resources/color.png</normaloff>:/Resources/color.png</iconset>
   </property>
   <property name="text">
    <string>字体颜色</string>
   </property>
  </action>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>显示辅助线</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>closeAction</sender>
   <signal>triggered()</signal>
   <receiver>CreatorWnd</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
